clusterloader2/pkg/measurement/common/simple/scheduler_latency.go

```diff
@@ -60,25 +60,25 @@ func (s *schedulerLatencyMeasurement) Execute(config *measurement.MeasurementCon
 	if err != nil {
 		return summaries, err
 	}
-	provider, err := util.GetStringOrDefault(config.Params, "provider", config.ClusterConfig.Provider)
+	provider, err := util.GetStringOrDefault(config.Params, "provider", config.ClusterFramework.GetClusterConfig().Provider)
```

Comment:
NIT: If you did this refactoring  (s/config.ClusterConfig/config.ClusterFramework/GetClusterConfig/g) in a separate PR it would be much easier to review.

Probably not worth doing it know but keep it in mind for future.

Reply:
Ok.